### DISCUSSION TITLE
Can visitors write a post in blog?

### DISCUSSION THREAD

**Discussion by kdg5424** [squidfunk/mkdocs-material#7678] [Q&A]
Hello Thank you for your wonderful theme.

I have a question about blog plugin.
I want to make a blog or board that visitors can write a post (maybe mainly Q&A post).
If I add the blog plugin in my page, can visitors wirte the posts?
Or is there any way to make it?

Best regards

Donggyu Kim

**Comment by alexvoss** [comment]
If a Q&A is what you have in mind then perhaps what you want is to add a discussion system. There is an [integration for Giscus](https://squidfunk.github.io/mkdocs-material/tutorials/blogs/engage/?h=discussion#add-a-discussion-system) and it would be possible to come up with similar integrations.

Re the blog plugin: this produces static pages just as you would expect from something that builds on top of a static site generator.

**Comment by kamilkrzyskow** [comment]
Neither MkDocs nor the theme aims to be a web-based CMS, like Wordpress. There is no account management or online access etc.
For people to add new posts they would have to access the files on their local PCs / some Codespace and modify the files under `blog_dir/posts`. Basically set up your git repository, and manage the docs with GitHub or GitLab, like the [theme does it](https://github.com/squidfunk/mkdocs-material/tree/master/docs).

But like Alex said, there might be an external service like Giscus for comments, but instead of comments you'd do so for that whole post creation. However, I doubt there is something like that, because it would need to integrate with MkDocs to manage accounts, and the underlying files 🤔 

As for Giscus, it requires the users to have an GitHub account, so you need to ask yourself, or your future prospective blog visitor, what sort of account do they have, and if they're willing to register on GitHub when they don't use it at all.

**Comment by kdg5424** [comment]
Thank you guys.
I'll consider to implement Giscus to my page.
